src/main/java/org/kuali/kfs/pdp/batch/service/impl/Iso20022FormatExtractor.java

```diff
@@ -879,8 +898,15 @@ private PostalAddress6 constructPostalAddress(
             isoCountryCode = wasPaymentGroupCreatedPriorToISOCountryConversion(templatePaymentGroup)
                     ? convertFIPSCountryValueToISOCountryCode(countryValue)
                     : convertISOCountryValueToISOCountryCode(countryValue);
+        }  else if (stateAndPostalCodeAreWithinUnitedStates(templatePaymentGroup.getState(),
```

Comment:
is there an extra space here (before "else if")?

Reply:
Good catch, I'll fix that.